felwithea/#Galrun_Starforge.lua

```diff
@@ -1,7 +1,14 @@
+-- epic expedition version of Struggles within the Progeny
+-- Game Update Notes: April 19, 2017
+-- Valdoon Paladin Epic 2.0 (Raid) - Lowered the minimum number of players necessary to request this raid from 10 to 6.
+local paladin_minimum_players = 10
+local paladin_expedition = { "The Asylum of Invoked Stone", paladin_minimum_players, 54 }
+local mmcc_dz = { "mmcc", 50, eq.seconds("3h") }
+mmcc_dz.compass = { "lfaydark", 3877.23, -127.446, -53.598 }
+mmcc_dz.safereturn = { "lfaydark", 3847, -56, -50, 0 }
+mmcc_dz.zonein = { -424.0, -108.0, 1.25, 0 }
```

Comment:
These are readable enough; I know that these are coordinates and can assume enough that this is x, y, z, h but it would be nice if this was key: value for readability

As long as examples are clearly documented and we make an effort to simplify readability where we can we'll be good